list_sorting.py

```diff
@@ -0,0 +1,39 @@
+#function to sort elements according to evens, odds and characters
+def list_sort(a):
+    if isinstance(a, list):
+        #list categories
+        word_cat = ['evens','odds','chars']
+        #sorted lis
+        sorted_dictionary = {}
+        char_list = []
+        evens_list = [] 
+        odds_list = []
+        for x in a:
+            if isinstance(x, str):
+                char_list.append(x)
+                
+            elif x % 2 == 1:
+                evens_list.append(x)
+                
+            elif x % 2 == 0:
+                odds_list.append(x)
+            else:
+                return False
+
+        sorted_dictionary[word_cat[2]] = char_list
+        char_list.sort()
+        sorted_dictionary[word_cat[1]] = evens_list
+        evens_list.sort()
+        sorted_dictionary[word_cat[0]] = odds_list
```

Comment:
This can be put on one line e.g
```py
sorted_dictionary[word_cat[0]] = odds_list.sort()
```

Reply:
When I do this sorted_dictionary[word_cat[0]] = odds_list.sort() and run the program the output is this:
{'evens': None, 'chars': None, 'odds': None}